packages/test/local-server-stress-tests/src/stressDataObject.ts

```diff
@@ -0,0 +1,320 @@
+		void this.context.containerRuntime
+			.createDataStore(
+				asChild
+					? [...this.context.packagePath, StressDataObject.factory.value.type]
+					: StressDataObject.factory.value.type,
+			)
+			.then(async (dataStore) => {
+				const maybe: FluidObject<StressDataObject> | undefined =
+					await dataStore.entryPoint.get();
+				assert(maybe?.StressDataObject !== undefined, "must be stressDataObject");
+				this.defaultStressObject.registerLocallyCreatedObject({
+					type: "stressDataObject",
+					handle: dataStore.entryPoint,
+					tag,
+					stressDataObject: maybe.StressDataObject,
+				});
+			});
+	}
+}
+export type ContainerObjects =
+	| { type: "newBlob"; handle: IFluidHandle; tag: `blob-${number}` }
+	| {
+			type: "stressDataObject";
+			tag: `datastore-${number}`;
+			handle: IFluidHandle;
+			stressDataObject: StressDataObject;
+	  };
+
+export class DefaultStressDataObject extends StressDataObject {
+	public static readonly alias = "default";
+
+	public get DefaultStressDataObject() {
+		return this;
+	}
+
+	/**
+	 * these are object created in memory by this instance of the datastore, they
+	 * will also be in  these the containerObjectMap, but are not necessarily usable
+	 * as they could be detached, in which can only this instance can access them.
+	 */
+	private readonly _locallyCreatedObjects: ContainerObjects[] = [];
+	public async getContainerObjects(): Promise<readonly Readonly<ContainerObjects>[]> {
+		const globalObjects: Readonly<ContainerObjects>[] = [...this._locallyCreatedObjects];
+		const containerRuntime = // eslint-disable-next-line import/no-deprecated
+			this.context.containerRuntime as IContainerRuntimeWithResolveHandle_Deprecated;
+		for (const url of this.containerObjectMap.keys()) {
+			// the container objects map will see things before they are attached,
+			// so they may not be available to remote clients yet.
+			// Additionally, there is no way to observe when an
+			// object goes from detached to attached.
+			// Due to the both the above, we need to always try
+			// to resolve each object, and just ignore those which can't
+			// be found.
+			const resp = await containerRuntime.resolveHandle({
+				url,
+				headers: { [RuntimeHeaders.wait]: false },
+			});
+			if (resp.status === 200) {
+				const maybe: FluidObject<IFluidLoadable & StressDataObject> | undefined = resp.value;
+				const handle = maybe?.IFluidLoadable?.handle;
+				if (handle !== undefined) {
+					const entry = this.containerObjectMap.get<ContainerObjects>(url);
+					switch (entry?.type) {
+						case "newBlob":
+							globalObjects.push({
+								...entry,
+								handle,
+							});
+							break;
+						case "stressDataObject":
+							assert(maybe?.StressDataObject !== undefined, "must be stressDataObject");
+
+							globalObjects.push({
+								type: "stressDataObject",
+								tag: entry.tag,
+								handle,
+								stressDataObject: maybe.StressDataObject,
+							});
+							break;
+						default:
```

Comment:
Not immediately clear if this default no-op is intentional, maybe add a comment or assert.